js/src/forum/components/SuspendUserModal.js

```diff
@@ -36,53 +43,10 @@ export default class SuspendUserModal extends Modal {
         <div className="Form">
           <div className="Form-group">
             <label>{app.translator.trans('flarum-suspend.forum.suspend_user.status_heading')}</label>
-            <div>
-              <label className="checkbox">
-                <input type="radio" name="status" checked={!this.status()} value="" onclick={withAttr('value', this.status)} />
-                {app.translator.trans('flarum-suspend.forum.suspend_user.not_suspended_label')}
-              </label>
-
-              <label className="checkbox">
-                <input
-                  type="radio"
-                  name="status"
-                  checked={this.status() === 'indefinitely'}
-                  value="indefinitely"
-                  onclick={withAttr('value', this.status)}
-                />
-                {app.translator.trans('flarum-suspend.forum.suspend_user.indefinitely_label')}
-              </label>
-
-              <label className="checkbox SuspendUserModal-days">
-                <input
-                  type="radio"
-                  name="status"
-                  checked={this.status() === 'limited'}
-                  value="limited"
-                  onclick={(e) => {
-                    this.status(e.target.value);
-                    m.redraw.sync();
-                    this.$('.SuspendUserModal-days-input input').select();
-                    e.redraw = false;
-                  }}
-                />
-                {app.translator.trans('flarum-suspend.forum.suspend_user.limited_time_label')}
-                {this.status() === 'limited' ? (
-                  <div className="SuspendUserModal-days-input">
-                    <input
-                      type="number"
-                      min="0"
-                      value={this.daysRemaining()}
-                      oninput={withAttr('value', this.daysRemaining)}
-                      className="FormControl"
-                    />
-                    {app.translator.trans('flarum-suspend.forum.suspend_user.limited_time_days_text')}
-                  </div>
-                ) : (
-                  ''
-                )}
-              </label>
-            </div>
+            <div>{this.radioItems().toArray()}</div>
+          </div>
+          <div className="Form-group">
+            <div>{this.formItems().toArray()}</div>
           </div>
```

Comment:
sorry, I need to improve my communication skills, but what I exactly meant was:

```jsx
           <div className="Form">
              {this.formItems().toArray()}
          </div>
```

and then formItems would have the radio form group and the message form group and the reason form group, that's how forms are organized across flarum's UI

```js
formItems() {
    items.add('radio-stuff..', (
        <div className="Form-group">
            <div>{this.radioItems().toArray()}</div>
        </div>
    ));

    items.add('reason', (
        <div className="Form-group">
            <label>...</label>
            <textarea ...>...</textarea>
        </div>
    ));

    items.add('message', (
        <div className="Form-group">
            <label>...</label>
            <textarea ...>...</textarea>
        </div>
    ));
}
```

Reply:
Ahhhh, sorry, I think that was my misunderstanding actually. No problem, I'll get that changed after I sort the `localstorage` change :)